sphinxcontrib/confluencebuilder/publisher.py

```diff
@@ -510,10 +529,7 @@ def make_connection(self, host):
         # build connection
         if self.https:
             try:
-                context = None
-                if self.disable_ssl_validation:
-                    import ssl
-                    context = ssl._create_unverified_context()
+                context = self._setup_ssl_context()
```

Comment:
there is still a condition where user would want a non-verified SSL connection surely?

Reply:
This is still supported. If they specify to disable ssl support, we can still return a valid ssl context with `check_hostname` set to false and the `verify_mode` set to `ssl.CERT_NONE`.